src/riscv-integration.adoc

```diff
@@ -801,18 +801,23 @@ xref:mtval2-cheri-causes[xrefstyle=short] respectively.
 .Encoding of TYPE field
 [#mtval2-cheri-type,width=65%,float="center",align="center",options=header,cols="30%,70%"]
 |==============================================================================
-| CHERI Type Code           | Description
-| {cheri_excep_type_pcc}    | CHERI instruction fetch fault
-| {cheri_excep_type_data}   | CHERI data fault due to load, store or AMO
-| {cheri_excep_type_jump}   | CHERI jump or branch fault
-| 3-15                      | Reserved
+| CHERI Type Code             | Description
+| {cheri_excep_type_pcc}      | CHERI instruction fetch fault
+| {cheri_excep_type_data}     | CHERI data fault due to load, store or AMO
+| {cheri_excep_type_jump}     | CHERI jump or branch fault
+| {cheri_excep_type_lpf}      | CHERI <<cheri_pte_ext,PTE>> load page fault^1^
+| {cheri_excep_type_spf_crg0} | CHERI <<cheri_pte_ext,PTE>> store page fault^1^ (PTE.CRG=0)
+| {cheri_excep_type_spf_crg1} | CHERI <<cheri_pte_ext,PTE>> store page fault^1^ (PTE.CRG=1)
+| 6-15                        | Reserved
 |==============================================================================
 
+^1^ if {cheri_pte_ext_name} is implemented
+
 .Encoding of CAUSE field
 [#mtval2-cheri-causes,width=55%,float="center",align="center",options=header]
 |==============================================================================
 | CHERI Cause Code              | Description
-| {cheri_excep_cause_tag}       | Tag violation
+| {cheri_excep_cause_tag}       | Tag violation, or TYPE is a <<cheri_pte_ext,PTE>> fault
```

Comment:
Oh no, you don't want to lump this under CHERI faults. It's still a page fault, just with extra information.

Reply:
this just means that the field isn't populated for the PTE faults

Reply:
the TYPE field is all you need, and CAUSE=0 if it's a PTE fault, so it's redundant

Reply:
reworked